docs/profiles.md

```diff
@@ -301,3 +304,35 @@ The following command runs the tests and applies the secrets specified in `profi
     $ inspec exec examples/profile-attribute --attrs examples/profile-attribute.yml
 
 See the full example in the InSpec open source repository: https://github.com/chef/inspec/tree/master/examples/profile-attribute
+
+# Profile files
+
+An InSpec profile may contain additional files that can be accessed during tests. This coveres use-cases where e.g. a list of ports is provided to be tested.
+
+To access these files, they must be stored in the `files` directory at the root of a profile. They are accessed by their name relative to this folder with `inspec.profile.file(...)`.
+
+Here is an example for reading and testing a list of ports. The folder structure is:
```

Comment:
Funny enough, I have taken the ports exactly from a customer example 😁 I also see why they want to use it that way and it's ok from my perspective.

I also agree with your premise! I wanted to avoid parsing a file to keep it really simple, but it might not hurt that much. I'll take your example, thank you for sharing.